README.md

```diff
@@ -31,13 +31,13 @@ See also (related specs):
 
 ## Install
 
-Requires Node.js 8.3+
+Requires Node.js 10+
```

Comment:
Only testing with 12+, should this match?  package.json should perhaps have matching engine field as well.

Reply:
Thanks, fixed.